### PR TITLE
xtask

### PR DESCRIPTION
**Pull request by Emilgardis**

### CODE REVIEW

**Review by Emilgardis — COMMENTED** [review]
bors r+

**Comment by Emilgardis** [comment]
bors r-

one more bug

File: .cargo/config.toml [hunk 1]
```diff
@@ -1,3 +1,4 @@
 [alias]
-cross-dev = ["run", "--bin", "cross-dev", "--features", "dev", "--"]
-build-docker-image = ["cross-dev", "build-docker-image"]
\ No newline at end of file
+xtask = ["run", "-p", "xtask", "--"]
+cross-dev = ["xtask"]
```
Comment (Alexhuszagh) [inline .cargo/config.toml:3]: Shouldn't this be `cross-xtask`?

[the remark this answers is not in the document]
Comment (Emilgardis) [reply .cargo/config.toml:3]: I'll remove it 👍🏽

**Review by Alexhuszagh — APPROVED** [review]

**Comment by Alexhuszagh** [comment]
Can I merge or do you want to squash the commits into 1?

**Comment by Emilgardis** [comment]
I have no preference

**Pull request merged**